<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manual_window</class>
 <widget class="QWidget" name="manual_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIPCALs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QPushButton" name="continue_button">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="return_button">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>837</width>
        <height>984</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="1" colspan="4">
        <widget class="QLineEdit" name="target_line"/>
       </item>
       <item row="6" column="0" colspan="5">
        <widget class="QLabel" name="caliboptions_lbl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CALIBRATION OPTIONS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="17" column="4">
        <widget class="QLineEdit" name="priorant_line"/>
       </item>
       <item row="25" column="4">
        <widget class="QComboBox" name="chanout_line"/>
       </item>
       <item row="13" column="4">
        <widget class="QCheckBox" name="subarray_chck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="disk_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Disk number</string>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <widget class="QCheckBox" name="loadal_chck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="22" column="4">
        <widget class="QLineEdit" name="minsolint_line"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="filepath_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Filepath</string>
         </property>
        </widget>
       </item>
       <item row="23" column="4">
        <widget class="QLineEdit" name="maxsolint_line"/>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="output_line"/>
       </item>
       <item row="19" column="4">
        <widget class="QLineEdit" name="maxrefantscans_line"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="output_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Output directory</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="calsour_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Calibrator source</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="disk_line"/>
       </item>
       <item row="26" column="4">
        <widget class="QLineEdit" name="edgeflag_line"/>
       </item>
       <item row="26" column="2">
        <widget class="QLabel" name="edgeflag_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Edge flagging</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="5">
        <widget class="QLabel" name="refantoptions_lbl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>REFERENCE ANTENNA OPTIONS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QCheckBox" name="caliball_chck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="3">
        <widget class="QLabel" name="caliball_lbl">
         <property name="text">
          <string>Calibrate all</string>
         </property>
        </widget>
       </item>
       <item row="12" column="4">
        <widget class="QLineEdit" name="freqsel_line"/>
       </item>
       <item row="14" column="0" colspan="3">
        <widget class="QLabel" name="shift_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Phase center shift</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="userno_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User number</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="freqsel_lbl">
         <property name="text">
          <string>Frequency selection</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLabel" name="phasref_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Phase ref calibrator</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="5">
        <widget class="QPushButton" name="more_options_btn">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>More Options</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="target_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Target</string>
         </property>
        </widget>
       </item>
       <item row="9" column="4">
        <widget class="QLineEdit" name="phasref_line"/>
       </item>
       <item row="11" column="0" colspan="3">
        <widget class="QLabel" name="loadall_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load all sources</string>
         </property>
        </widget>
       </item>
       <item row="18" column="4">
        <widget class="QCheckBox" name="centrant_chck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="21" column="4">
        <widget class="QLineEdit" name="solint_line"/>
       </item>
       <item row="13" column="0" colspan="3">
        <widget class="QLabel" name="subarray_lbl">
         <property name="text">
          <string>Look for subarray</string>
         </property>
        </widget>
       </item>
       <item row="14" column="4">
        <widget class="QLineEdit" name="shift_line"/>
       </item>
       <item row="28" column="2">
        <widget class="QLabel" name="interactive_lbl">
         <property name="text">
          <string>Interactive plots</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="5">
        <widget class="QLabel" name="loadoptions_lbl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LOAD OPTIONS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="16" column="4">
        <widget class="QLineEdit" name="refant_line"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="selectfile_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>          Select file          </string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="28" column="4">
        <widget class="QCheckBox" name="interactive_chck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="userno_line"/>
       </item>
       <item row="7" column="2" colspan="3">
        <widget class="QLineEdit" name="calsour_line"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="filepath_line"/>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="selectdir_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select folder</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="3">
        <widget class="QLabel" name="refant_lbl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Reference antenna</string>
         </property>
        </widget>
       </item>
       <item row="20" column="0" colspan="5">
        <widget class="QLabel" name="fringoptions_lbl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FRINGE FIT OPTIONS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="24" column="0" colspan="5">
        <widget class="QLabel" name="exportoptions_lbl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>EXPORT OPTIONS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="27" column="0" colspan="5">
        <widget class="QLabel" name="plotoptions_lbl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PLOT OPTIONS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="3">
        <widget class="QLabel" name="priorant_lbl">
         <property name="text">
          <string>Priority antennas</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0" colspan="3">
        <widget class="QLabel" name="centrant_lbl">
         <property name="text">
          <string>Search central antennas</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0" colspan="3">
        <widget class="QLabel" name="maxrefantscans_lbl">
         <property name="text">
          <string>Maximum scans</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0" colspan="3">
        <widget class="QLabel" name="solint_lbl">
         <property name="text">
          <string>Fixed solution interval</string>
         </property>
        </widget>
       </item>
       <item row="22" column="0" colspan="3">
        <widget class="QLabel" name="minsolint_lbl">
         <property name="text">
          <string>Minimum solution interval</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0" colspan="3">
        <widget class="QLabel" name="maxsolint_lbl">
         <property name="text">
          <string>Maximum solution interval</string>
         </property>
        </widget>
       </item>
       <item row="25" column="1" colspan="2">
        <widget class="QLabel" name="chanout_lbl">
         <property name="text">
          <string>Channel out</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
